docs/proposals/central-segmented-registry.md

```diff
@@ -0,0 +1,144 @@
+# Central Segmented Registry
+
+## Introduction and Use Case
+
+Customers have expressed a need for multiple OpenShift deployments to be able to reference a central registry. They wish to deploy this registry globally to serve a distributed workforce.
+
+Customers have also asked for recommendations for how to segment this registry so images are separated for different *application* lifecycle environments (e.g. dev, test, prod). A common but unsatisfactory approach is to prefix project namespaces with these environment labels. The [Image Promotion](image-promotion.md#areas-of-improvement) proposal acknowledges gaps in this approach.
+
+Today, OpenShift deployments support a single integrated registry, referenced as service **docker-registry** in the **default** namespace. All other registries are "external". This proposal suggests a way to achieve more flexibility in referencing registries.
+
+## Goals
+
+* **Consistent Image Reference** We want users to consistently reference the same image name across all of their clusters as part of a promotion workflow. We want to do this through controlling environment assignment at the cluster level. In other words, the production cluster should only have access to images in the production registry.
```

Comment:
This needs to take into account cluster federation (ubernetes) as well.  Multiple clusters is really a federation problem, because security has to be unified across them as well.